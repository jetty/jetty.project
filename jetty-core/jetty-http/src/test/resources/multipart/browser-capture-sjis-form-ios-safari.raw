------WebKitFormBoundaryj1Xj6oPRT7sp3VPE
Content-Disposition: form-data; name="japanese"

Œ’Ž¡
------WebKitFormBoundaryj1Xj6oPRT7sp3VPE
Content-Disposition: form-data; name="hello"

ƒƒƒ†&#25094;ƒ^
------WebKitFormBoundaryj1Xj6oPRT7sp3VPE--
