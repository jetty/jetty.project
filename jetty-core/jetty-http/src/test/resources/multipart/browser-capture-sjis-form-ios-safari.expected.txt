Request-Header|Accept|text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Request-Header|Accept-Encoding|gzip, deflate
Request-Header|Accept-Language|en-us
Request-Header|Connection|keep-alive
Request-Header|Content-Length|249
Request-Header|Content-Type|multipart/form-data; boundary=----WebKitFormBoundaryj1Xj6oPRT7sp3VPE
Request-Header|Host|192.168.0.119:9090
Request-Header|Origin|http://192.168.0.119:9090
Request-Header|Referer|http://192.168.0.119:9090/sjis-form.html
Request-Header|Upgrade-Insecure-Requests|1
Request-Header|User-Agent|Mozilla/5.0 (iPad; CPU OS 11_2_6 like Mac OS X) AppleWebKit/604.5.6 (KHTML, like Gecko) Version/11.0 Mobile/15D100 Safari/604.1
Parts-Count|2
Part-ContainsContents|japanese|健治
Part-ContainsContents|hello|ャユ戆タ